sdk/python/foundation-models/benchmarking/evaluating_oai_models.ipynb

```diff
@@ -0,0 +1,199 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3e4c5468-7585-481e-bb20-ad91373cac6c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import load_job\n",
+    "\n",
+    "# load the pipeline from the yaml def\n",
+    "pipeline_job = load_job(f\"./evaluation_pipelines/azure_openai/{eval_name}.yaml\")\n",
+    "\n",
+    "# Set pipeline job inputs\n",
+    "pipeline_job.settings.default_compute = compute_name\n",
+    "pipeline_job.inputs.endpoint_url = aoai_endpoint_url\n",
+    "pipeline_job.inputs.ws_connection_name = connection_name\n",
+    "pipeline_job.inputs.sample_ratio = sample_ratio\n",
+    "\n",
+    "# Start the job in the Workspace\n",
+    "returned_job = ml_client.jobs.create_or_update(\n",
+    "    pipeline_job, experiment_name=experiment_name\n",
+    ")\n",
+    "returned_job"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "095005d9-1deb-4097-a718-4059a335ec09",
+   "metadata": {},
+   "source": [
+    "Run the next cell to stream the job. Notebook execution will be paused until the job finishes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1376aa46-725d-4d6c-8725-3a2d4d52bdfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wait until the job completes\n",
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07fa2c79-f875-4c4f-8f09-37158b56bdcd",
+   "metadata": {},
+   "source": [
+    "## Retrieve accuracy scores from the run\n",
+    "When the pipeline finishes, you can retrieve evaluation metrics from the run via mlflow. The primary measure of accuracy for the evals is `mean_exact_match`, with the exception of human_eval which uses `pass@1`. \n",
+    "\n",
+    "Mean exact match is the proportion of model predictions that exactly match the corresponding correct answers. Thus, it is applicable to question answering evaluations that are multiple choice or have a single, correct answer. The pass@1 metric is used for evaluating code generation and is the proportion of model generated code solutions that pass a set of unit tests given in the eval dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "299bf8ea-49ab-44c3-96e6-51902fa03df4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "accuracy_metric_name = \"mean_exact_match\" if eval_name != \"human_eval\" else \"pass@1\"\n",
+    "\n",
+    "mlflow_tracking_uri = ml_client.workspaces.get(\n",
+    "    ml_client.workspace_name\n",
+    ").mlflow_tracking_uri\n",
+    "mlflow.set_tracking_uri(mlflow_tracking_uri)\n",
+    "\n",
+    "run = mlflow.get_run(run_id=returned_job.name)\n",
+    "metric_val = run.data.metrics[accuracy_metric_name]\n",
+    "\n",
+    "print(f\"Accuracy metric name: {accuracy_metric_name}\")\n",
+    "print(f\"Accuracy metric value: {metric_val}\")"
```

Comment:
If sampling_ratio != 1.0, can we add a warning in this cell of both the notebooks that recommends to use sampling_ratio=1.0 for proper results.

Reply:
Added